iree/modules/check/dialect/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ iree_cc_binary(
   DEPS
     iree::modules::check::dialect
     iree::tools::iree_opt_library
```

Comment:
modules/check isn't included at modules/ yet, but I don't think this will compile without a fix for https://github.com/google/iree/issues/653 (I have a fix ready for review, just FYI)

https://github.com/google/iree/blob/ff099d7789510ffe666b4961fa9201208e0b216b/iree/modules/CMakeLists.txt#L13-L16